CMakeLists.txt

```diff
@@ -21,10 +27,10 @@ endif()
 #####################################
 # Define Options
 #####################################
-option(CROW_BUILD_EXAMPLES     "Build the examples in the project"      ON )
-option(CROW_BUILD_TESTS        "Build the tests in the project"         ON )
+option(CROW_BUILD_EXAMPLES     "Build the examples in the project"      ${CROW_IS_MAIN_PROJECT})
+option(CROW_BUILD_TESTS        "Build the tests in the project"         ${CROW_IS_MAIN_PROJECT})
+option(CROW_INSTALL            "Add install step for Crow"              ${CROW_IS_MAIN_PROJECT})
```

Comment:
I think this option should be enabled by default. It doesn't really harm and installing Fetch Content Dependencies is a common requirement.

Reply:
I guess it works either way. I'll default it to `ON` then. 👍